src/Imath/ImathFrustum.h

```diff
@@ -581,6 +657,25 @@ Frustum<T>::normalizedZToDepth (T zval) const
     }
 }
 
+template <class T>
+IMATH_CONSTEXPR14 T
+Frustum<T>::normalizedZToDepth_noexcept (T zval) const noexcept
+{
+    T Zp = zval * 2.0 - 1;
```

Comment:
Similar, this should probably be T(2) and T(1) to avoid double conversion and int types

Reply:
+1

Reply:
The last push addresses this issue.